pkg/mount/scuzz/ata.go

```diff
@@ -134,6 +134,10 @@ func ataString(a []byte) string {
 }
 
 // unpackIdentify unpacks a wordBlock into an Info.
+// TODO: I am not sure what the status block is passed here and if it is really
+// necessary, this should be audited
+//
+//nolint:unparam
```

Comment:
so we have code we don't understand that we're patching up with no lint directives? Kinda nasty?